scrnaseq.Rmd

```diff
@@ -91,16 +93,41 @@ param$mart_attributes = c(param$annot_main,
 # Prefix of mitochondrial genes 
 param$mt = "^MT-"
 
-# The number of PCs to use; adjust this parameter based on JackStraw and Elbowplot 
-param$pc_n = 10
+# Filters
+param$cell_filter = list(nFeature_RNA=c(200,NA), percent_mt=c(NA,20))
+param$features_filter = list(min_counts=1, min_cells=1) # feature has to be found by at least one count in one cell
+param$samples_to_drop = c("NC", "RNA") # cells from these samples will be dropped after initial QC
 
 # Whether or not to remove cell cycle effects
-param$cc_remove = FALSE
+param$cc_remove = TRUE
```

Comment:
Any particular reason you want to set this default to TRUE?

Reply:
no

Reply:
Then back to FALSE in my opinion. :-)

Reply:
Done